pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module.adoc

```diff
@@ -0,0 +1,20 @@
+include::../attributes.adoc[]
+[id='module_{context}']
+= Module
+
+A module, in modular documentation, is defined as topic-specific structured informational content written in the specified syntax, stored in a file, that can be published on its own and as a part of a larger document in {ProductShortName}.
+
+The following are the properties of a module:
+
+* Conforms to the module syntax.
+* Self-contained: A module contains all the necessary information about a topic required by a user to achieve a goal. The goal of a user can be to understand a concept, to perform a task, or to find reference information.
+* Standalone: A module does not require any surrounding text to be useful to the reader; a module is publishable on its own.
+* Pluggable: Amodule can be used unmodified as a part of a larger document.
+
+Types of modules defined in modular documentation:
+
+* Concept module
```

Comment:
We already stated these are types of modules, so we should be fine to just list the type (concept, procedure, reference).